### ISSUE TITLE
Add basic TS typings

### ISSUE THREAD

**Issue by james-camilleri** [robinscholz/svelte-inline-svg#25]
This package throws up errors when using Svelte with TypeScript, I was wondering if it would be possible to include basic typings for the component. I've prepared a very simple set, it shouldn't be too much hassle to add.

```ts
import { SvelteComponentTyped } from 'svelte'

export declare class InlineSvgProps {
    src: string
    transformSrc: (src: SVGElement) => SVGElement
    [attribute: string]: any
}

export default class InlineSvg extends SvelteComponentTyped<InlineSvgProps> {
}
```

**Comment by dennismphil** [comment]
I was unable to get typescript support on import. 

What worked for me was I created a file `svelte-inline-svg.d.ts` and declared the types manually. 

Marked the `transformSrc` attribute as optional

```ts
// file: src/types/svelte-inline-svg.d.ts
declare module 'svelte-inline-svg' {
  import { SvelteComponentTyped } from 'svelte';
  export declare class InlineSvgProps {
    src: string;
    transformSrc?: (src: SVGElement) => SVGElement;
    [attribute: string]: unknown;
  }
  export default class InlineSvg extends SvelteComponentTyped<InlineSvgProps> { }
}

```

**Comment by certainlyakey** [comment]
> For anyone stumbling with TS support for this module, I was unable to get typescript support by default on import.

Just had the same issue - VSCode would highlight the statement `import InlineSVG from 'svelte-inline-svg' as an error and svelte-inline-svg as a non-installed package. Thanks for the fix, it worked!

**Comment by robinscholz** [comment]
Reopening this … will look into it when I have some time!

**Comment by robinscholz** [comment]
This should finally be fixed with 1.1.4. Please open a new issue should anyone run into more problems.

**Comment by robinscholz** [comment]
@james-camilleri Thanks for persevering
